docs/en/sql-reference/statements/create/quota.md

```diff
@@ -11,19 +11,25 @@ Syntax:
 
 ``` sql
 CREATE QUOTA [IF NOT EXISTS | OR REPLACE] name [ON CLUSTER cluster_name]
-    [KEYED BY {'none' | 'user name' | 'ip address' | 'forwarded ip address' | 'client key' | 'client key or user name' | 'client key or ip address'}]
+    [KEYED BY {USER_NAME | IP_ADDRESS | CLIENT_KEY | CLIENT_KEY, USER_NAME | CLIENT_KEY, IP_ADDRESS} | NOT KEYED]
```

Comment:
The command `SHOW CREATE QUOTA` displays the parameters `USER_NAME`, ... and `RESULT_BYTES`, ... in lower case. The command `CREATE QUOTA` can be written with those parameters in upper case, but lower case looks a bit better.

Reply:
@vitlibar We have a guide - all key words must be written in upper case...